centreon/src/Core/ServiceTemplate/Infrastructure/Repository/DbWriteServiceTemplateRepository.php

```diff
@@ -40,6 +43,238 @@ public function __construct(DatabaseConnection $db)
+            \PDO::PARAM_INT
+        );
+        $statement->bindValue(':command_id', $newServiceTemplate->getCommandId(), \PDO::PARAM_INT);
+        $statement->bindValue(
+            ':command_arguments',
+            $this->serializeArguments($newServiceTemplate->getCommandArguments())
+        );
+        $statement->bindValue(':event_handler_id', $newServiceTemplate->getEventHandlerId(), \PDO::PARAM_INT);
+        $statement->bindValue(
+            ':event_handler_arguments',
+            $this->serializeArguments($newServiceTemplate->getEventHandlerArguments())
+        );
+        $statement->bindValue(
+            ':acknowledgement_timeout',
+            $newServiceTemplate->getAcknowledgementTimeout(),
+            \PDO::PARAM_INT
+        );
+        $statement->bindValue(
+            ':is_activated',
+            $newServiceTemplate->isActivated()
+        );
+        $statement->bindValue(
+            ':is_locked',
+            $newServiceTemplate->isLocked(),
+            \PDO::PARAM_INT
+        );
+        $statement->bindValue(
+            ':active_checks_enabled',
+            (string) YesNoDefaultConverter::toInt($newServiceTemplate->getActiveChecks())
+        );
+        $statement->bindValue(
+            ':event_handler_enabled',
+            (string) YesNoDefaultConverter::toInt($newServiceTemplate->getEventHandlerEnabled())
+        );
+        $statement->bindValue(
+            ':flap_detection_enabled',
+            (string) YesNoDefaultConverter::toInt($newServiceTemplate->getFlapDetectionEnabled())
+        );
+        $statement->bindValue(
+            ':check_freshness',
+            (string) YesNoDefaultConverter::toInt($newServiceTemplate->getCheckFreshness())
+        );
+        $statement->bindValue(
+            ':notifications_enabled',
+            (string) YesNoDefaultConverter::toInt($newServiceTemplate->getNotificationsEnabled())
+        );
+        $statement->bindValue(
+            ':passive_checks_enabled',
+            (string) YesNoDefaultConverter::toInt($newServiceTemplate->getPassiveCheck())
+        );
+        $statement->bindValue(
+            ':volatility',
+            (string) YesNoDefaultConverter::toInt($newServiceTemplate->getVolatility())
+        );
+        $statement->bindValue(
+            ':low_flap_threshold',
+            $newServiceTemplate->getLowFlapThreshold(),
+            \PDO::PARAM_INT
+        );
+        $statement->bindValue(
+            ':high_flap_threshold',
+            $newServiceTemplate->getHighFlapThreshold(),
+            \PDO::PARAM_INT
+        );
+        $statement->bindValue(
+            ':max_check_attempts',
+            $newServiceTemplate->getMaxCheckAttempts(),
+            \PDO::PARAM_INT
+        );
+        $statement->bindValue(
+            ':description',
+            $newServiceTemplate->getName()
+        );
+        $statement->bindValue(
+            ':comment',
+            $newServiceTemplate->getComment()
```

Comment:
No use of repositoryTrait->legacyHtmlEncode for string parameters ?

Reply:
No, comments are not encoded in database.